src/pop_chain.erl

```diff
@@ -1,15 +1,16 @@
--module(pop_protocol).
+-module(pop_chain).
 
 -export([
-	 init/5,
-	 add_block/3,
+	 new/1,
+	 add_block_in_order/3,
 	 add_transaction/2,
 	 generate_new_block/2,
 
 	 find_block_by_id/2,
+	 get_genisys_block/1,
```

Comment:
genesis 🥔

Reply:
i literally misspell this word every time i type it

good thing i figured out how to do **replace in all files** in emacs 😱

Reply:
:O someday I'll learn how to replace in all files in emacs.